README.md

```diff
@@ -3,336 +3,93 @@
-`write`. Every claim is linked to an `option` token.
-
-```solidity
-struct Claim {
-        uint256 option;
-        uint112 amountWritten;
-        uint112 amountExercised;
-        bool claimed;
-    }
-```
-
-##### Option
-
-The `Option` struct contains all data about an option chain/token and is keyed on the
-unique hash `keccak256(abi.encode(Option memory))` where `settlementSeed` is set to
-`0` at the time of hashing.
-
-```solidity
-    struct Option {
-        address underlyingAsset;
-        uint40 exerciseTimestamp;
-        uint40 expiryTimestamp;
-        address exerciseAsset;
-        uint96 underlyingAmount;
-        uint160 settlementSeed;
-        uint96 exerciseAmount;
-    }
-```
-
-##### Underlying
-
-The `Underlying` struct contains information about the underlying assets for 1
-wei of a given token ID in the settlement engine.
-
-```solidity
-struct Underlying {
-        address underlyingAsset;
-        int256 underlyingPosition;
-        address exerciseAsset;
-        int256 exercisePosition;
-    }
-```
+3. Run `forge test` (this will install dependencies, build the project's smart contracts, and run the tests on a local fork of mainnet)
+
+## Security Information
+- Audit info TODO
+- Bug bounty info TODO
+- Security contact info TODO
+
+## Protocol Specification
+- [Introduction](#introduction-to-the-protocol)
+- [Trust Model](#trust-model)
+- [Documentation](TODO)
+- [Cucumber Features](./test/features)
+- Glossary _(coming soon)_
+
+### Introduction to the Protocol
+The core of Valorem is the Settlement Engine, which follows the [ERC-1155 multi-token](https://eips.ethereum.org/EIPS/eip-1155) standard. Options can be written for any pair of valid ERC-20 assets (excluding rebasing, fee-on-transfer, and ERC-777 tokens). When written, an options contract is represented by semi-fungible Option tokens, which can be bought/sold/transferred between addresses like any ERC-1155 token.
+
+The option writer's claim to the underlying asset(s) (if not exercised) and exercise asset(s) (if exercised) is represented by a non-fungible Option Lot Claim token. This Claim NFT can be redeemed for their share of the underlying plus exercise assets, based on current. The Settlement Engine uses a internal day bucketing approach, to facilitate fair assignment while minimizing O(n) complexity and gas costs.
+
+The structure of an options contract is as follows:
+
+- **Underlying asset:** the ERC-20 address of the asset to be received
+- **Underlying amount:** the amount of the underlying asset contained within an option contract of this type
+- **Exercise asset:** the ERC-20 address of the asset needed for exercise
+- **Exercise amount:** the amount of the exercise asset required to exercise this option
+- **Earliest exercise timestamp:** the timestamp after which this option may be exercised
+- **Expiry timestamp:** the timestamp before which this option must be exercised
```

Comment:
Slight nitpick: option need not be exercised, but language implies it must be

Reply:
Good catch. updated both timestamp lines to use 'can' as verb